src/styles/blocks/clock-face.scss

```
@@ -0,0 +1,35 @@
+.watch-center {
+  @extend %watch-center;
+}
+
+.clock-face {
+  @include circle($circle-size);
+  @extend %clock-face-centering;
+
+  border: 0.3vh solid black;
+  background-color: $c-bg-color;
+
+  &__hand {
+    position: absolute;
+    top: 15vh;
+    height: 35%;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    transform-origin: bottom;
+  }
+}
+
+.hand-minute {
+  animation: rotate 360s infinite;
```

Wrong transition-duration

```suggestion
  animation: rotate 3600s infinite;
```

---

this is incorrect, because we have only two clock-hand (second and minute). so, second-hand - 60s, minute-hand - 360s

---

1 minute = 60 seconds
60 minutes = 60 * 60 seonds => 3600 seconds

There's no sense to use 360s (6 minutes) because we won't be able to see how many minutes passed since the activation of the stopwatch

> this is incorrect, because we have only two clock-hand (second and minute). so, second-hand - 60s, minute-hand - 360s

---

Haha, really. Thats my fall